### PR TITLE
【Hackathon No.31】 Fix divide zero bug for Case 6:paddle.nn.functional.softmax_with_cross_entropy

### PR DESCRIPTION
**Pull request by zeroRains**
### PR types

Bug fixes
### PR changes

APIs
### Describe

[#49919(comment)](https://github.com/PaddlePaddle/Paddle/issues/49919#issuecomment-1386643789)

### CODE REVIEW

**Comment by zeroRains** [comment]
image
我就改了两行，但是只命中一行CI过不了怎么办嘛

**Comment by luotao1** [comment]
> 我就改了两行，但是只命中一行CI过不了怎么办嘛

说明补充的单测并没有进入 if 条件里面

**Comment by zeroRains** [comment]
image
我在本地跑单例的时候是可以覆盖进去的呀

File: python/paddle/nn/functional/loss.py [hunk 1]
```diff
@@ -2338,6 +2338,8 @@ def softmax_with_cross_entropy(
             # Tensor(shape=[1], dtype=float32, place=Place(gpu:0), stop_gradient=True,
             #        [1.15328646])
     """
+    if 0 in logits.shape:
+        raise ValueError("The dims of Input(X) should be greater than 0.")
```
Comment (wanghuancoder) [inline python/paddle/nn/functional/loss.py:2342]: 你好，这个判断还是加到C++kernel里吧。原则是：能在C++里加的检查不在python加。否则影响动态图API性能

Comment (zeroRains) [reply python/paddle/nn/functional/loss.py:2342]: ok，我改一下

Comment (zeroRains) [reply python/paddle/nn/functional/loss.py:2342]: image
额，好像这个不用修了，那我取消PR吧

**Comment by luotao1** [comment]
> 额，好像这个不用修了，那我取消PR吧

好的
